_positions/visual-designer.md

```diff
@@ -27,3 +27,5 @@ Visual designers at 18F craft highly usable and beautiful web-based experiences
 - Regularly participate in group critiques, serving as both “critic” and the “critiqued.”
 - Scour the web for applicable new design solutions and share findings with the team.
 - Support the 18F brand by contributing to in-house projects.
+
+##[Apply to join 18F]({{ site.baseurl }}/pages/apply.html)
```

Comment:
@rooneywp Same questions as in #207. Conceivably we could have two application streams simultaneously, a general one at `/pages/apply.html` and a role-specific (set of) one(s) hosted on Lever; if that's what you intend in this set of PRs, let's chat about it.